ts/nni_manager/training_service/reusable/trialDispatcher.ts

```diff
@@ -45,13 +52,15 @@ class TrialDispatcher implements TrainingService {
     private enableVersionCheck: boolean = true;
 
     private trialConfig: TrialConfig | undefined;
-    private runnerSettings: RunnerSettings;
-
-    private commandEmitter: EventEmitter | undefined;
-    private commandChannel: CommandChannel | undefined;
 
     private readonly trials: Map<string, TrialDetail>;
     private readonly environments: Map<string, EnvironmentInformation>;
+    // make public for ut
+    public environmentServiceList: EnvironmentService[] = [];
+    public commandChannelDict: Map<Channel, CommandChannel>;
```

Comment:
It should be in environmentService itself. All use of this map is related to environment service. Explained in above comments, in some cases, it may not be a single instance. Maintain such a map in this level is not necessary and not a good practice also.

Reply:
updated.